controller/app/controllers/BaseController.java

```diff
@@ -478,6 +478,12 @@ private static void handleBackwardCompatibility(
       response.getParams().setErr("USER_CONSENT_NOT_FOUND");
       response.getParams().setStatus("USER_CONSENT_NOT_FOUND");
     }
+    if (request.path() != null
+            && (request.path().startsWith("/v1/user/get/:idType/:id"))
```

Comment:
this check should be  like this (request.path().startsWith("/v1/user/get/")) not ("/v1/user/get/:idType/:id"))

Reply:
ok